src/applications/personalization/profile360/components/BankInfoForm.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
+
+import { ACCOUNT_TYPES_OPTIONS } from '../constants';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    routingNumber: {
+      type: 'string',
+      pattern: '^\\d{9}$',
+    },
+    accountNumber: {
+      type: 'string',
+      pattern: '^\\d{1,17}$',
+    },
+    accountType: {
+      type: 'string',
+      enum: Object.values(ACCOUNT_TYPES_OPTIONS),
+    },
+  },
+  required: ['accountNumber', 'routingNumber', 'accountType'],
+};
+
+export const uiSchema = {
+  routingNumber: {
+    'ui:title':
+      'Routing number (Your 9-digit routing number will update your bank’s name)',
+    'ui:errorMessages': {
+      pattern: 'Please enter the bank’s 9-digit routing number.',
+      required: 'Please enter the bank’s 9-digit routing number.',
+    },
+  },
+  accountNumber: {
+    'ui:title': 'Account number (No more than 17 digits)',
+    'ui:errorMessages': {
+      pattern: 'Please enter your account number.',
+      required: 'Please enter your account number.',
+    },
+  },
+  accountType: {
+    'ui:title': 'Account type',
+    'ui:errorMessages': {
+      required: 'Please select the type that best describes the account.',
+    },
+  },
+};
+
+const BankInfoForm = ({
+  cancelButtonClasses,
```

Comment:
Do we need this prop `cancelButtonClasses`? It doesn't seem like we are passing it in?

Reply:
yes! sorry I thought I left a comment about that. But we need it for Profile 2.0 where we make the `cancel` button look like a text link via the `va-button-link` class

Reply:
oh yeah, I'd made a much a longer JS comment and then pulled it out but meant to add the info in GitHub. Good eye, btw.